migration/transitioning.rst

```diff
@@ -0,0 +1,92 @@
+.. _transitioning_mt:
+
+Migrating an existing single-node PostgreSQL database to Citus sometimes requires adjusting the schema and queries for optimal performance. Citus extends PostgreSQL with distributed functionality, but it is not a drop-in replacement that scales out all workloads. A performant Citus cluster involves thinking about the data model, tooling, and choice of SQL features used.
+
+Migration tactics differ between the two main Citus use cases of multi-tenant applications and real-time analytics. The former requires fewer data model changes so we'll begin there.
+
+Multi-tenant Data Model
+=======================
+
+Citus is well suited to hosting B2B multi-tenant application data. In this model application tenants share a Citus cluster and a schema. Each tenant's table data is stored in a shard determined by a configurable tenant id column. Citus pushes queries down to run directly on the relevant tenant shard in the cluster, spreading out the computation. Once queries are routed this way they can be executed without concern for the rest of the cluster. These queries can use the full features of SQL, including joins and transactions, without running into the inherent limitations of a distributed system.
+
+This section will explore how to model for the multi-tenant scenario, including necessary adjustments to the schema and queries.
+
+The general process is
+
+1. Recreate the schema of the original database on the Citus coordinator node.
+2. Modify the schema for multi-tenant distribution.
+3. Copy the data from the original database to Citus.
```

Comment:
As discussed, I think we can remove this section. This PR isn't a step-by-step guide yet.